container-signing/action.yaml

```
@@ -0,0 +1,42 @@
+name: "Container signing"
+description: "Cosign based action to create container signatures."
+
+inputs:
+  docker-digest:
+    description: "Set the digest from the docker build and push action e.g the output of steps.build-and-push.outputs.digest."
+    required: true
+  docker-tags:
+    description: "Set the tags from the docker meta action e.g the output of steps.meta.outputs.tags."
+    required: true
+  cosign-key-password:
+    description: "Set the cosign key password, it not set a keyless signature will be created."
+    required: false
+  cosign-key:
+    description: "Set the cosign key, it not set a keyless signature will be created."
+    required: false
+
+branding:
+  icon: "package"
+  color: "green"
+
+runs:
+  using: "composite"
+  steps:
+    - uses: sigstore/cosign-installer@main
```

IMHo we should use the latest release version.
```suggestion
    - uses: sigstore/cosign-installer@v3
```